app/package.json

```diff
@@ -107,6 +112,9 @@
     "websocket-extensions": "^0.1.4",
     "yargs-parser": "^18.1.3"
   },
+  "resolutions": {
```

Comment:
Why was this added?

Reply:
It was not supposed to be added. Will remove.